blockchainetl/jobs/exporters/composite_item_exporter.py

```diff
@@ -27,20 +27,29 @@
 
 
 class CompositeItemExporter:
-    def __init__(self, filename_mapping, field_mapping=None):
+    def __init__(self, filename_mapping, field_mapping=None, field_to_export=None):
```

Comment:
Can we simplify this and just use field_mapping without field_to_export? In blocks_and_transactions_item_exporter.py you can pass the schema to field_mapping, or if not provided by the user the default fields.

Reply:
apologies for being so long to reply, i chose to keep it seperate because the later sections of the code, the field_to_export are intersected with the field_mapping to ensure that only valid fields are kept and that original order is maintained